api/model.go

```diff
@@ -114,6 +114,7 @@ type Backup struct {
 	VolumeSize             string               `json:"volumeSize"`
 	VolumeCreated          string               `json:"volumeCreated"`
 	VolumeBackingImageName string               `json:"volumeBackingImageName"`
+	OwnerID                string               `json:"ownerID"`
```

Comment:
Do we need this internal info? If people want to know, users can check the backup resource directly

Reply:
I'm exposing the `OwnerID` for automation e2e.
https://github.com/longhorn/longhorn-tests/pull/986
```
    backup_volume, backup = find_backup(client, volume.name, snapshot.name)
    assert backup.ownerID != original_ownerID, \
        f'expecting {backup.name} ownerID {original_ownerID} to change'

```

Reply:
Removed. https://github.com/longhorn/longhorn-tests/pull/986#discussion_r889940530